en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -99,85 +87,83 @@ Activities and Organisations Schemas
 Codelist Changes
 ================
 
+A full diff of all **embedded** codelist changes in 2.03 can be found in the `IATI GitHub <https://github.com/IATI/IATI-Codelists/compare/version-2.02...version-2.03#files_bucket>`__
+
 Embedded to Non-Embedded:
 ------------------------
 
-The following codelists have moved from Embedded to Non-Embedded (`#114 <https://github.com/IATI/IATI-Codelists/issues/114>`__) (`Discussion link <https://discuss.iatistandard.org/t/redefine-selected-codelists-as-non-embedded-included-2-03/854>`__):
-
-.. list-table::
-   :widths: 16 16 16
-   :header-rows: 0
-   
-   * -	ActivityScope
-	    -	BudgetIdentifier
-	    -	BudgetIdentifierSector-Category
-   * -	BudgetIdentifierSector
-   	  -	BudgetIdentifierVocabulary
-	    -	CRSAddOtherFlags
-   * -	ConditionType
-	    -	ContactType
-	    -	DescriptionType
-   * -	DisbursementChannel
-   	  -	DocumentCategory-Category
-	   	-	GeographicExactness
-   *	-	GeographicLocationClass
-	   	-	GeographicLocationReach
-	   	-	GeographicVocabulary
-   *	-	GeographicalPrecision
-	   	-	IndicatorMeasure
-	   	-	LoanRepaymentPeriod
-   *	-	LoanRepaymentType
-	   	-	OrganisationType
-	   	-	OtherIdentifierType
-   *	-	PolicyMarker
-	   	-	PolicyMarkerVocabulary
-	   	-	PublisherType
-   *	-	RegionVocabulary
-	   	-	ResultType
-	   	-	SectorVocabulary
-   *	-	TiedStatus
-	   	-	ResultType
-	   	-	VerificationStatus
-   *	-	PublisherType
-	   	-	PolicyMarkerVocabulary
-	   	-	PolicyMarker
-   *	-	RegionVocabulary
-	   	-	TiedStatus
-     	- - 
+Alongside 2.03, the following codelists have moved from Embedded to Non-Embedded (`#114 <https://github.com/IATI/IATI-Codelists/issues/114>`__) (`Discussion link <https://discuss.iatistandard.org/t/redefine-selected-codelists-as-non-embedded-included-2-03/854>`__):
```

Comment:
Should this list have nested points like the list for the table? If so this is possible in rst format:

http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#bullet-lists